src/System.CommandLine/Symbol.cs

```diff
@@ -80,10 +80,6 @@ public IEnumerable<Symbol> Parents
         /// <summary>
         /// Gets completions for the symbol.
         /// </summary>
-        public IEnumerable<CompletionItem> GetCompletions() => 
-            GetCompletions(CompletionContext.Empty());
```

Comment:
This feels like an unnecessary breaking change to me.

Reply:
> This feels like an unnecessary breaking change to me.

The main point of my changes is simplification of the APIs which should help us get them approved and moved into BCL.

One of the users raised a concern about removing this method: https://github.com/dotnet/command-line-api/issues/1891#issuecomment-1307575196 but by exposing `CompletionContext.Empty` we are going to make it testable.

Reply:
Ok that's fair, if is proven later on that this method is valuable, we can bring it back.